src/styles/molecules/_molecules.navigation.scss

```diff
@@ -142,6 +143,11 @@
           }
         }
       }
+      
+      > button {
```

Comment:
Is deze aanvulling nodig?
Volgens mij wordt die al uitgevoerd omdat `m-nav-tabs` extend op `m-nav-list`.

Reply:
blijkbaar niet, afgestemd met jasper, ik zal het verwijderen